ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml

```diff
@@ -181,25 +187,44 @@ SettingsPageLayout {
         id: collectibleView
 
         CommunityCollectibleView {
+            id: collectibleViewComponent
+
             viewWidth: root.viewWidth
             preview: d.preview
             holdersModel: root.holdersModel
 
             onMintCollectible: {
                 d.collectibleName = name
-                root.mintCollectible(artworkSource,
-                                     name,
-                                     symbol,
-                                     description,
-                                     supply,
-                                     infiniteSupply,
-                                     transferable,
-                                     selfDestruct,
-                                     chainId,
+                Global.openPopup(signTransactionDialog,
```

Comment:
Is it necessary to use `Global.openPopup`? It makes the solution more problematic for testing (page for CommunityMintTokensSettingsPanel doesn't handle popup properly). Maybe it would be enough to define the popup locally and open via standard api.

Reply:
We need to have a global reference to open the poupup in the center of the application. If we create it locally, it will just be centered inside the component and will not have the look we are looking for.
If there is another way to do that, please, share it!

Reply:
`anchors.centerIn: Overlay.overlay` ! solved with local popup!